recipe/meta.yaml

```diff
@@ -20,28 +24,37 @@ build:
     - fairseq-score = fairseq_cli.score:cli_main
     - fairseq-train = fairseq_cli.train:cli_main
     - fairseq-validate = fairseq_cli.validate:cli_main
-  script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [win]
+  script:
+    # compiling Cython token_block_utils_fast.pyx module so it can be imported
+    - {{ PYTHON }} setup.py build_ext --inplace
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    # - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --global-option="build_ext" --global-option="--inplace"
+  missing_dso_whitelist:
+    - $RPATH/libtorch_cpu.so         # [linux]
+    - $RPATH/libtorch_cpu.dylib      # [osx]
+    - $RPATH/libtorch_python.so      # [linux]
+    - $RPATH/libc10.so               # [linux]
+    - $RPATH/libtorch_python.dylib   # [osx]
+    - $RPATH/libc10.dylib            # [osx]
```

Comment:
did you verify that these files are actually there?

Reply:
We sure did for `pytorch`.  I can check `libc10.dylib` once again.

Reply:
fine by me

Reply:
I just checked, they're all there.

Reply:
thanks!